conans/client/remover.py

```diff
@@ -96,10 +99,15 @@ def _remote_remove(self, ref, package_ids, remote):
             tmp = self._remote_manager.remove_packages(ref, package_ids, remote)
             return tmp
 
+    @staticmethod
+    def _message_removing_editable(ref):
+        return "Package '{r}' is installed as editable, unlink it first using " \
+               "command 'conan link {r} --remove'".format(r=ref)
+
     def _local_remove(self, ref, src, build_ids, package_ids):
         if self._cache.installed_as_editable(ref):
-            raise ConanException("Package '{r}' is installed as editable, unlink it first using "
-                                 "command 'conan link {r} --remove'".format(r=ref))
+            self._user_io.out.warn(self._message_removing_editable(ref))
```

Comment:
Why converting this error to warning? Seems unrelated to this PR.

Reply:
Uou, maybe something wrong in a conflict

Reply:
Now I remember, I also completed the remove with editables test. If you are removing by pattern, you probably shouldn't fail if some matching reference is in editable, you want to ignore it, and delete the rest.